### DISCUSSION TITLE
Output Pixels of specific grain

### DISCUSSION THREAD

**Discussion by antonyEng** [mtex-toolbox/mtex#1451] [Ask Anything]
hello, 
I have succeded in selecting a grain, and I want to get all the Euler angles for pixels constructing the grain, how?

please help !!

**Comment by antonyEng** [comment]
lets want to study grain id  800 in urs data 
use code :  c=ebsd(grains(800)) 
this put the ebsd data in this variable c 
then to get the euler angles  : 
c.rotations.phi1
c.rotations.phi
c.rotations.phi1
take care these results in radian . 
i am happy to be able to answer my question ,

**Comment by euerka** [comment]
Very limited experience with AZTecCrystal as follows:
1. Select the specific grain that you want to analyze
2. Create a subset of selected grain
3. New a project for this subset
4. Extract all the Euler angles for pixels of the selected grain

I am not sure whether you can use the same steps at MTEX or not.

Hopeful, it helps a bit.

**Comment by lfgmorales** [comment]
Like this:

https://mtex-toolbox.github.io/SelectingGrains.html

?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 4 Jul 2022, at 18:01, euerka ***@***.***> wrote:

﻿

Very limited experience with AZTecCrystal as follows:

  1.  Select the specific grain that you want to analyze
  2.  Create a subset of selected grain
  3.  New a project for this subset
  4.  Extract all the Euler angles for pixels of the selected grain

I am not sure whether you can use the same steps at MTEX or not.

Hopeful, it helps a bit.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1451#discussioncomment-3079207>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6GZSQDLWNXP3POGKDVSMDGZANCNFSM5YLHXMUA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1451#discussioncomment-3079207", "url": "https://github.com/mtex-toolbox/mtex/discussions/1451#discussioncomment-3079207", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
